cli/utils.go

```diff
@@ -93,6 +95,24 @@ func mustGetContextConfig(cmd *cobra.Command) *viper.Viper {
 	return cmd.Context().Value(cfgContextKey).(*viper.Viper) //nolint:forcetypeassert
 }
 
+// MessageChans contains message channels that can be used to relay important information
+// after calling `cli.Start`.
+type MessageChans struct {
```

Comment:
suggestion: I don't think I like defining this in the utils.go file (as that really doesn't scale), and I don't think I like the idea of lots of non-command files in the `cli` package, leaving me with the suggestion of moving this into a `message.go` file in a new `utils` sub-package of `cli`.